packages/transport/src/core/Transport.ts

```diff
@@ -56,6 +58,7 @@ export class Transport {
     }
 
     private static readonly defaultConfig: IConfig = {
+        allowIdentityCreation: false,
```

Comment:
I would vote for leaving the functionality as-is by using true as a default and using false in the connector